### ISSUE TITLE
Dynamically expand iid schemas with wildcards

### ISSUE THREAD

**Issue by jbusecke** [leap-stc/cmip6-leap-feedstock#44]
We currently [ask the user](https://github.com/leap-stc/cmip6-leap-feedstock#how-can-i-request-new-data) to provide a list of fully specified iids. 

But using pangeo-forge-esgf, we can already expand iids with wildcards ([example](https://github.com/jbusecke/pangeo-forge-esgf#parsing-a-list-of-instance-ids-using-wildcards)). I think it would be very nice for users to provide a list of iids with wildcards, which then gets expanded automatically. 

Realistically this should only be enabled after #43 to avoid reprocessing a lot of already existing data.

This was mentioned in #17 but I opted to make a new issue to track this separately from the dynamic url querying.
